dev/devicelab/README.md

```diff
@@ -48,28 +54,20 @@ Notably, it will start and stop Gradle, for instance.
 
 ### Running all tests
 
-To run all tests defined in `manifest.yaml`, use option `-a` (`--all`):
-
-```sh
-../../bin/cache/dart-sdk/bin/dart bin/run.dart -a
-```
-
-This defaults to only running tests supported by your host device's platform
-(`--match-host-platform`) and exiting after the first failure (`--exit`).
+`TODO(chillers): Update to use .ci.yaml. https://github.com/flutter/flutter/issues/74660`
 
 ### Running specific tests
```

Comment:
test_runner.dart currently does not support this. Should this be ported over or removed?